tests/garage/logger/test_logger.py

```diff
@@ -0,0 +1,94 @@
+import unittest
+
+from garage.logger import CsvOutput, logger, tabular, TextOutput
```

Comment:
a simpler way to organize these tests is as unit tests rather than integration tests.

i.e. you can write a small test for each of your `*Output` classes which verifies it produces the correct output given an input from the logger, which you manually construct rather than using the `Logger` to produce. then you can write a single small test for `Logger` using a single output (or a dummy output) which verifies simple things (e.g. add/remove, passes prefixes correctly etc.) but doesn't have to examine any output.

Reply:
you might find the library `unittest.mock` helpful